docs/cave/bundles-and-procedures.md

```diff
@@ -51,15 +51,15 @@ To save a product display to a path within the file directory of the workstation
 ### New Procedure
 
 * Select the menu **File > Procedures > New**
-* Select **Copy Into** to add all loaded resources to the Procedure Stack
+* Select **Copy Into** to add all loaded resources from your current map to the Procedure Stack
 * Select **Save** (or **Save As**) and then enter a name for the Procedure before clicking **OK** to save.
 
-![image](../images/O925hqJNac.gif)
+![image](../images/save-procedure.gif)
 
 ### Open Procedure
 
 Similar to creating a new Procedure, select **File > Procedures > Open**, select the saved resources and click **Load** to load them to CAVE.
```

Comment:
Add section breaks (`---`) between these last few sections, to be consistent with the rest of the page.